### PR TITLE
Fix typo and rearrange words in IDLE news item 

### PR DESCRIPTION
**Pull request by GeeTransit**

### CODE REVIEW

**Review by eamanu — APPROVED** [review]

File: Misc/NEWS.d/next/IDLE/2019-08-04-17-10-01.bpo-36419.TJZqOc.rst [hunk 1]
```diff
@@ -1 +1 @@
-IDLE - Refactor autocompete and improve testing.
+Refactor autocomplete and improve testing.
```
Comment (aeros) [inline Misc/NEWS.d/next/IDLE/2019-08-04-17-10-01.bpo-36419.TJZqOc.rst]:
```suggestion
Refactor IDLE autocomplete and improve testing.
```

**Review by aeros — CHANGES_REQUESTED** [review]
Thanks for the PR @GeeTransit.

I agree with the typo correction, but it should still be mentioned that this change involves the IDLE.

File: Misc/NEWS.d/next/IDLE/2019-08-04-17-10-01.bpo-36419.TJZqOc.rst (hunk 1 above)
Comment (GeeTransit) [reply Misc/NEWS.d/next/IDLE/2019-08-04-17-10-01.bpo-36419.TJZqOc.rst]: Yup, this sounds good too!

**Review by aeros — APPROVED** [review]
Thanks for making the recommended changes.

**Review by raulcd — APPROVED** [review]

**Comment by terryjreedy** [comment]
I mis-hit the merge button while editing the title, such that Miss I. could not report success here.  I reverted and will make a new PR.

**Pull request merged**